AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1844,6 +1844,7 @@ masfrandy.com##.berlangganan-box
 infomoney.com.br##.infomoney-newsletter
 donbosco-medien.de###wh_nl_hint
 gamespot.com##.nlsub
+newseveryday.com##.f-subscribe
```

Comment:
@stanislav-atr If I'm not wrong, we do not block those kind of elements if it's not annoying, for example in footer.
https://github.com/AdguardTeam/AdguardFilters/pull/80327#issuecomment-821090769

By the way, it seems that share buttons from this report https://github.com/AdguardTeam/AdguardFilters/issues/80487#issuecomment-820439874 are not blocked.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/115385291-43f3ef80-a1d8-11eb-935f-c7ee6c0d397c.png)

</details>

Reply:
Fixed social btns.

There are a lot of rules, that remove newsletters from footers though.

Reply:
>There are a lot of rules, that remove newsletters from footers though.

Okay then.